plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/ReentrantBoundedExecutor.java

```diff
@@ -30,7 +30,7 @@ public class ReentrantBoundedExecutor
     private final Executor boundedExecutor;
     private final Executor coreExecutor;
 
-    ReentrantBoundedExecutor(Executor coreExecutor, int maxThreads)
```

Comment:
What is the point of this change? If this is related to the next commit then I would squash it.